rerun-failed-run/README.md

```diff
@@ -0,0 +1,66 @@
+# rerun-failed-run
+
+This reusable workflow can be used to retrigger in GitHub Actions workflows to automatically retrigger them if they failed and contain a specific error message in their build logs.
+
+## Usage
+
+The workflow can be invoked with the aid of the GitHub CLI from another workflow. In case the calling workflow does not belong in the same repository the designated GitHub App Token should be generated in order to allow the invocation. For that purpose, a github app is already set up, and its credentials can be found in the following Vault secret paths:
+
+```
+secret/data/products/infra/ci/retrigger-gha-workflow RETRIGGER_APP_KEY;
+secret/data/products/infra/ci/retrigger-gha-workflow RETRIGGER_APP_ID;
+```
+
+You can customize how many times you want the workflow to retrigger your job by configuring the number of attempts in the job level. See th example provided
+below for more information.
+
```

Comment:
I'd also add a disclaimer that the retry limit must always be specified (otherwise there's a risk or re-running the original workflow forever)